Scripts/validate-library-evolution.swift

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env xcrun swift
+
+import Foundation
+
+/**
+ This script builds a custom swift package containing all frameworks inside of the xcarchive passed when launching.
+ 
+ Input parameters:
+  - CommandLine.arguments[1]: Path to the .xcarchive that contains all the frameworks that should be tested on library-evolution support
+  - CommandLine.arguments[2]: Name of the project that allows testing (Has to be a name, not a path as it's reused as the scheme of the swift package)
+ */
+
+let fileManager = FileManager.default
+let currentDirectoryURL = URL(fileURLWithPath: fileManager.currentDirectoryPath, isDirectory: true)
+
+let xcarchiveURL = URL(fileURLWithPath: CommandLine.arguments[1], relativeTo: currentDirectoryURL)
+let outputProjectURL = URL(fileURLWithPath: CommandLine.arguments[2], relativeTo: currentDirectoryURL)
+
+let dSYMsDirectoryURL = xcarchiveURL.appending(path: "dSYMs")
+let frameworksDirectoryURL = xcarchiveURL.appending(path: "Products/Library/Frameworks")
+
+// MARK: - Convenience Methods
+
+func infoPlistContent(for frameworkName: String) -> String {
+    return """
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>AvailableLibraries</key>
+    <array>
+        <dict>
+            <key>BinaryPath</key>
+            <string>\(frameworkName).framework/Adyen</string>
+            <key>DebugSymbolsPath</key>
+            <string>dSYMs</string>
+            <key>LibraryIdentifier</key>
+            <string>ios-arm64_x86_64-simulator</string>
+            <key>LibraryPath</key>
+            <string>\(frameworkName).framework</string>
+            <key>SupportedArchitectures</key>
+            <array>
+                <string>arm64</string>
+                <string>x86_64</string>
+            </array>
+            <key>SupportedPlatform</key>
+            <string>ios</string>
+            <key>SupportedPlatformVariant</key>
+            <string>simulator</string>
+        </dict>
+    </array>
+    <key>CFBundlePackageType</key>
+    <string>XFWK</string>
+    <key>XCFrameworkFormatVersion</key>
+    <string>1.0</string>
+</dict>
+</plist>
+"""
+}
+
+func packageFileContent(for frameworkNames: [String], scheme: String) -> String {
+    
+    let dependencies = frameworkNames.map { ".byName(name: \"\($0)\")"}.joined(separator: ",\n")
+    let binaryTargets = frameworkNames.map { ".binaryTarget(name: \"\($0)\", path: \"\($0).xcframework\")"}.joined(separator: ",\n")
+    
+    return """
+// swift-tools-version: 5.8
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "\(scheme)",
+    platforms: [.iOS(.v13)],
```

Comment:
Should this be 12 as we still support it?

Reply:
Fair - the binary is compiled for iOS 12 but of course it makes sense to use it here as well 👌